### PR TITLE
Kelsy wells

### PR DESCRIPTION
**Pull request by kelsywells**
Finished product with added cards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
...
 
+An event is a signal that something has happened within the website, for instance, a user has interacted with something on the page.
+
 3. What is an event listener?
 
+An event listener is a piece of code that waits for a given event to take place and returns a given response.
+
 4. Why would we convert a NodeList into an Array?
 
+A NodeList is too long to be listed out/an array makes it cleaner and easier to work with.
```
Comment (ari-mercado) [inline README.md:47]: Also, converting a NodeList into an Array allows us to use Array methods such as *map*, *filter*, and *find*.

File: components/Tabs/Tabs.js [hunk 2]
```diff
@@ -1,64 +1,67 @@
...
-    /* <- Delete this comment block when you work on the if statement
+
     // Check to see if this.tabData is equal to 'all'
-    if(){
+    if(this.tabData = 'all'){
+     
+   
       // If `all` is true, select all cards regardless of their data attribute values
-      // this.cards = ;
+      this.cards = document.querySelector('cards');
```
Comment (ari-mercado) [inline components/Tabs/Tabs.js:18]: Here we want a nodeList, so we can use *querySelectorAll* instead. Also, the argument should be *.card*

File: components/Tabs/Tabs.js [hunk 3]
```diff
@@ -1,64 +1,67 @@
...
-    // tabs.forEach()
+    tabs.forEach(item => item.classList.remove('active-tab'));
 
     // Select all of the elements with the .card class on them
-    // const cards = ;
+    const cards = document.querySelectorAll('.card');
 
     // Iterate through the NodeList setting the display style each one to 'none'
-    // cards.forEach()
+    cards.forEach(cards, () => display.style = 'none');
```
Comment (ari-mercado) [inline components/Tabs/Tabs.js:43]: The correct syntax here is:
    *cards.forEach(card => card.display.style = 'none');*

**Review by ari7946 — APPROVED** [review]
## Code Review Checklist

- [x] Follows Lambda Git Flow 
- [x] Frequent, clear, and descriptive commits
- [ ] No errors in the console
- [x] Code is written by the student
- [x] Code is sufficiently commented
- [x] Code is organized and easy to read
- [x] MVP Complete
- [ ] At least 1 stretch completed

### Notes:
Nice work, Kelsy! I like the number of descriptive commits. There's an error in the console due to a minor bug.

**Pull request merged**